src/strategy.coffee

```diff
@@ -13,19 +13,25 @@ module.exports = (oAuth2Strategy) ->
       options.authorizationURL ?= constants.defaultAuthorizationURL
       options.tokenURL ?= constants.defaultTokenURL
       options.verifyURL ?= constants.defaultVerifyURL
+      @params = new Object()
+      @params.scope = options.scope or ''

```

Comment:
passport-oauth2 expects `params` to be an options bag (see below, re:  `Oauth2Strategy.authorizationParams`)